indexes/base.js

```diff
@@ -40,8 +40,10 @@ module.exports = function makeBaseIndex(privateIndex) {
 
     processRecord(record, seq, pValue) {
       const buf = record.value
-      const author = bipf.decode(buf, bipf.seekKey2(buf, pValue, BIPF_AUTHOR, 0))
-      const sequence = bipf.decode(buf, bipf.seekKey2(buf, pValue, BIPF_SEQUENCE, 0))
+      const pValueAuthor = bipf.seekKey2(buf, pValue, BIPF_AUTHOR, 0)
+      const pValueSequence = bipf.seekKey2(buf, pValue, BIPF_SEQUENCE, 0)
+      const author = bipf.decode(buf, pValueAuthor)
+      const sequence = bipf.decode(buf, pValueSequence)
```

Comment:
These changes are unrelated to the PR, but when I ran `prettier`, the `decode` ended up taking many lines, so I refactored to create the `pValueAuthor` and keep each of these a one-liner.